scripts/bert/create_pretraining_data.py

```diff
@@ -416,15 +417,51 @@ def create_instances_from_document(x):
 
 MaskedLmInstance = collections.namedtuple('MaskedLmInstance',
                                           ['index', 'label'])
+def find_chinese_whole_word(universal_tokenizer, tokens, chs_tokenizer=None):
+    """find chinese characters in tokenized results, and tag them in BIO format with chinese "
+       tokenizer.
+    Parameters
+    ----------
+    tokenizer : BERTTokenizer
+    tokens : list of str
+        bert tokenized tokens: e.g.: ['你', '好', '12', '##3']
 
+    Returns
+    -------
+    A tuple of list: tokens, chs_token_bio;
+        e.g.:   tokens = ['你', '好', '12', '##3']
+                chs_token_bio = ['B', 'I', 'O', 'O']
+    """
```

Comment:
Move examples to the Example sections

```
def find_chinese_whole_word(universal_tokenizer, tokens, cn_tokenizer=None):
    """find chinese characters in tokenized results, and tag them in BIO format with chinese "
       tokenizer.

    Parameters
    ----------
    tokenizer : BERTTokenizer
        The tokenizer
    tokens : list of str
        bert tokenized tokens

    Returns
    -------
    A tuple of list: tokens, cn_token_bio

    Examples
    --------
    find_chinese_whole_word(tokenizer, ['你', '好', '12', '##3'], JiebaTokenizer())
    >>> ['你', '好', '12', '##3'], ['B', 'I', 'O', 'O']
```

Reply:
nit: rename `chs_` to `cn_` so that the naming is consistent with existing codebase?

Reply:
Done